### DISCUSSION TITLE
Interconnect vs libsigc++?

### DISCUSSION THREAD

**Discussion by ghost** [mosra/corrade#180] [General]
https://github.com/libsigcplusplus/libsigcplusplus

**Comment by mosra** [comment]
I've never used libsigc++ so I can't really tell the difference, but from a cursory look it has a lot more features. It also has a different, more restrictive license. 

The Interconnect library is deliberately kept very simple with just the essentials. So depending on whether you use other parts of Corrade already and want something lightweight, or whether you want a full feature set to handle complex application-side demands, pick one or the other. Nothing else in Corrade or Magnum relies on Interconnect so you're free to pick whichever fits best.
